docs/examples/edb/edb_example.ipynb

```diff
@@ -69,6 +69,29 @@
     "print(\"Didn't check the connection, no exception\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "4cfbfdf4",
+   "metadata": {},
+   "source": [
+    "### Connecting to Cloud DB\n",
+    "Alternatively, instead of connecting to your local MongoDB instance, you can connect to the cloud-hosted database for watertap. \n",
```

Comment:
```suggestion
    "Alternatively, instead of connecting to your local MongoDB instance, you can connect to the cloud-hosted database for WaterTAP. \n",
```